### ISSUE TITLE
Error while building app

### ISSUE THREAD

**Issue by Kallaf** [Ramotion/react-native-circle-menu#15]
This error appears while running `react-native run-android`  and the app cann't be built

error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `node_modules\@ramotion\react-native-circle-menu\node_modules\react-native\Libraries\react-native\react-native-implementation.js`: AccessibilityInfo could not be found within the project.

**Comment by danielaRiesgo** [comment]
Same here, does anyone know how to solve it?
